app/views/core_induction_programmes/lessons/_course_lesson_fields.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= f.govuk_error_summary %>
+<%= f.govuk_text_field :title, label: { text: "Lesson title" } %>
+<%= f.govuk_collection_select(
+      :course_module_id,
+      @course_modules,
+      :id,
+      :term_and_title,
+      label: { text: "Which module would you like the lesson to be assigned to?" },
+      ) %>
+<%= f.govuk_number_field(
+      :completion_time_in_minutes,
+      label: { text: "Time in minutes"},
+      width: 4, hint: { text: " For example, 90"},
+      value: @course_lesson.completion_time_in_minutes) %>
+<%= f.govuk_text_area(
+      :ect_summary,
+      label: { text: 'ECT summary' },
+      rows: 10) %>
+<%= f.govuk_text_area(
+      :mentor_summary,
+      label: { text: 'Mentor summary' },
+      rows: 10) %>
+<%= f.govuk_submit t(f.object.persisted? ? :update : :create, scope: [:helpers, :submit, :course_lesson]) %>
```

Comment:
From what I can tell `govuk_design_system_formbuilder` does not support the default i18n rails looks up so this has been added. if we can fix it upstream we remove this but keep the locale entries

Reply:
Personally I would have been tempted to have a `Save` text here. 

Or maybe to take the button out of the shared fields and type the version that we wanted in the actual template.

Buuut I don't actually mind it. We will probably spend a while discussing the use of translations on dev retro next week, in the meantime this can go in :)